codechain/crypto/ecdsa.py

```diff
@@ -0,0 +1,123 @@
+from coincurve import PrivateKey
+from coincurve import PublicKey
+from coincurve import verify_signature
+
+SECPK1_N = (
+    115792089237316195423570985008687907852837564279074904382605163141518161494337
+)
+
+# Gets ECDSA signature for message from private key.
+
+
+def sign_ecdsa(message, priv):
+    if not isinstance(message, (bytes, bytearray)) or len(message) != 32:
```

Comment:
To strictly follow the documents[1], these conditions should be separated. However, it would be tedious. So just let it be as is.

[1] https://docs.python.org/3/library/exceptions.html#TypeError

Reply:
I think it would be better to change it for the future.